examples/UbidotsSubscribeContext/UbidotsSubscribeContext.ino

```diff
@@ -0,0 +1,142 @@
+/****************************************
+ * Include Libraries
+ ****************************************/
+
+#include <math.h>
+#include <UbidotsMQTT.h>
+
+/****************************************
+ * Define Constants
+ ****************************************/
+
+#ifndef TOKEN
+#define TOKEN "token"  // Ubidots token
+#endif
+
+#define VARIABLE_LABEL "my-var"
+#define DEVICE_LABEL "my-dev"
+
+// Main loop execution period in seconds
+#define LOOP_PERIOD_SEC 5
+
+/****************************************
+ * Globals
+ ****************************************/
+
+UbidotsMQTT clientMQTT(TOKEN, callback);
+
+char ssid[32 + sizeof("")] = "";
+char bssid[2*6 + 5 + sizeof("")] = "";
+char context[64 + sizeof("")] = "";
+
+/****************************************
+ * User Functions
+ ****************************************/
+
+void callback(char* topic, uint8_t* payload, unsigned int length) {
+  /****************************************************************************
+   * Note: For the sake of simplicity, this callback only handles received
+   * payload as a serialized JSON object. Make sure you subscribe only to 
+   * complete dots, e.g. clientMQTT.ubidotsSubscribe("<device>", "<variable>", false),
+   * or make proper adjustments to support last value.
+   ****************************************************************************/
+  
+  Serial.println("--------------------------------");
+  
+  Serial.printlnf("Message from '%s': %.*s", topic, length, payload);
+  
+  char *jsonStr = (char *) malloc(length*sizeof(*payload) + sizeof(""));
```

Comment:
In our Library it is likely forbidden to use String objects, but the less we want in the examples files is to generate some kind of friction to the users when reading the code.

We do encourage whoever write these examples to write the code with the less traumatic functions. 

Long short story use String in .ino files.

Reply:
Hard to use String() in this case, will use strndup() to create a null-terminated string.